### PR TITLE
create the security page.

### LINKED ISSUE #938: create /security page
We need to update the layout of #915 
see https://deploy-preview-915--gitpod-kumquat.netlify.app/security

and add a link to the footer

Mockup:
Security
Thanks (1)

[Figma ](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=1443%3A1802)

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #938 
This is how it looks:

image

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/967"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/967"></a>

### CODE REVIEW

**Comment by mikenikles** [comment]
I see #937 as an another PR that claims to add a security page. Is #937 obsolete if we merge this one here?

File: src/contents/security.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"No packages or dependencies are downloaded to users' devices. Gitpod developer environments run in the cloud and are short-lived, protecting your local machines and other corporate resources from malicious attacks through execution of arbitrary code.",
+]
+
+export const programFeatures: ProgramFeature[] = [
+	{
+		title: 'Compliance',
+		paragraphs: ['Gitpod is a European company committed to developer and data privacy. We are GDPR compliant and provide our users with the ability to access and control the information that we collect about them.', 'Gitpod is built with security in mind and we continuously invest in security best practices. We are currently in the process of becoming SOC 2 compliant and you can request a copy of our SOC2 audit report as soon as it\'s available.'],
+		image: 'compliance.svg',
```
Comment (ChristinFrohne) [inline src/contents/security.ts:13]: is this the latest version of the GDPR icon? The alignment of the stars seems off

**Review by ChristinFrohne — COMMENTED** [review]
Great, thanks! How did you decide on the choice of logos? They are partly different from the mockup. Once we updated the landing page these logos should be in sync with that

Outstanding to-do's:
- [x] add a link to the footer

- [ ] adjust the mobile view
Bildschirmfoto 2021-09-15 um 09 44 38

**Comment by ChristinFrohne** [comment]
I thought #937 would be merged first and this Pr would update the layout. 
If we discard #937, this page (https://deploy-preview-937--gitpod-kumquat.netlify.app/security/report) needs to be included in this PR as well. Instead of the big call-out section at the end of /report, we should use the "thanks" section that is used before.
Could you add that @nisarhassan12 ?

File: src/contents/security.ts (hunk 1 above)
Comment (nisarhassan12) [reply src/contents/security.ts:13]: Nope. It was not I have updated it now.

**Comment by nisarhassan12** [comment]
> Could you add that @nisarhassan12 ?
@ChristinFrohne Done ✅  

I have also addressed the other feedback as well. Please have a look again. 

One thing to watch out for is the page titles that I added here https://github.com/gitpod-io/website/pull/967/commits/93a31cf887a348752d6bd73b38d43fea0fe139aa Thanks

**Comment by ChristinFrohne** [comment]
Thanks!
- sorry I should have been more clear about the "thanks" sections on /report. `security/report` should link to /`security/thanks` by reusing this section:
Bildschirmfoto 2021-09-15 um 13 49 51

- Did you see my former comment? "How did you decide on the choice of logos? They are partly different from the mockup. Once we updated the landing page these logos should be in sync with that"
- could you update the tab headline to "Security Disclosure Policy"
Bildschirmfoto 2021-09-15 um 13 55 22

Once you have addressed the feedback, please ping Geoff to give the messaging the final tweak.

**Comment by nisarhassan12** [comment]
> Did you see my former comment? "How did you decide on the choice of logos? They are partly different from the mockup. Once we updated the landing page these logos should be in sync with that"

Sorry I read that earlier but somehow it slipped from my mind, Now I have updated the order this is how it looks:

image

Please note that the first row now has the same order both on the Home and Security page.

**Comment by nisarhassan12** [comment]
> sorry I should have been more clear about the "thanks" sections on /report. security/report should link to /security/thanks by reusing this section:

Thanks. Done this is how it looked

image

which looks different than the rest of the white boxes we have in isolation on different pages so I have updated it to look like the following:

image

@ChristinFrohne Are you fine with this change ? Note that this change only applies to the report page and not the security page. Thanks

**Comment by ChristinFrohne** [comment]
Great initiative @nisarhassan12 ! Looks great ✨

**Review by ghuntley — APPROVED** [review]
Can't wait to have this shipped. Ready to go from my perspective.

**Comment by JohannesLandgraf** [comment]
It is really important that we take ownership seriously. We cannot merge company policies without the consent of the DRI for that topic. In this case this is @meysholdt who needs to approve this PR before it goes live.

**Pull request merged**